package.json

```diff
@@ -116,7 +116,8 @@
         "start": "lerna run --parallel start --ignore=docs",
         "build": "lerna run build --stream --no-private --ignore=@redhat-cloud-services/frontend-components-config*",
         "generate-translations": "npm run build && node packages/utils/src/mergeMessages.js",
-        "test": "jest",
+        "test:ct": "npx -w ./packages/advisor-components cypress run-ct",
```

Comment:
This will be changed in the future once other packages will start utilizing cypress-ct. For now, `-w` lets to run cypress tests in the advisor package.

Reply:
We don't want to drop jest. We will have to have two scripts. One for jest and one for cypress

Reply:
@Hyperkid123, I didn't propose to drop it, we will also use it in advisor-components. PTAL https://github.com/RedHatInsights/frontend-components/pull/1540/files/ce8082bc404e8e9027a2c8c52c783ee2de3c537a#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R120: I am just adding one more command on top of `jest` so that we can include cypress tests in `npm run test` command

Reply:
I will add more info soon with a brief overview of pros/cons/reasons, it is still work in progress

Reply:
Sorry I just looked at your comment and it looks like the te command is gone :face_exhaling: I was too lazy to look at the whole file :+1: